### PR TITLE
Migrate to the Android embedding v2

### LINKED ISSUE #200: Migrate App to Android Embedding v2
**Are you working on this issue? (Yes/No)** Yes

### PR DESCRIPTION
**Pull request by aman-singh7**
Fixes #200 

Describe the changes you have made in this PR -

Screenshots of the changes (If any) -

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -32,15 +32,18 @@ dependencies:
...
+  get: ^4.6.1
+  get_it: ^7.2.0
   google_sign_in: ^5.0.4
   html: ^0.15.0
   http: ^0.13.3
   intl: ^0.17.0
   mockito: ^5.0.17
   oauth2_client: ^2.2.2
-  photo_view: ^0.13.0
+  # photo_view: ^0.13.0
```
Comment (aman-singh7) [inline pubspec.yaml:43]: @manjotsidhu There are some bugs in the deployed version of this package but have been fixed on GitHub. So, I have temporarily updated this.

**Comment by Nitish145** [comment]
@aman-singh7 Can you break this PR into two of which the first handle the flutter 2.10.0's `Looking up a deactivated widget's ancestor is unsafe` error for `photo_view`. Then this one actually is `Migrating to Android embedding v2`.

**Comment by aman-singh7** [comment]
ok @Nitish145 :+1:

**Comment by aman-singh7** [comment]
@Nitish145 Done. Tests will pass only when both of them are merged. :sweat_smile:

**Comment by Nitish145** [comment]
@aman-singh7 Please rebase this PR, so that only Android embedding v2 changes are the ones that build this PR.

**Review by Nitish145 — APPROVED** [review]

**Pull request merged**